src/main/converter/opds.ts

```diff
@@ -176,6 +176,14 @@ export class OpdsFeedViewConverter {
             navigation = feed.Navigation.map((item) => {
                 return this.convertOpdsLinkToView(item);
             });
+
+            // concatenate all relative path to an absolute URL path
+            navigation = navigation.map((nav) => {
+                if (nav.url && !/[a-zA-Z]+\:\/\//.exec(nav.url)) {
```

Comment:
The `[a-zA-Z]+` RegExp pattern does not match the URL scheme syntax https://tools.ietf.org/html/rfc3986?#section-3.1 (`ALPHA *( ALPHA / DIGIT / "+" / "-" / "." )`)

In our use-case we only expect `http://` or `https://` (in my tests I have personally never come across anything else, not even `opds://` inside OPDS feeds), so instead of implementing the correct URL protocol syntax / RFC parsing rules, we could just match with `!/^https?:\/\//.exec(nav.url)`.

What do you think?

Reply:
sure

The opds:// is for attramenta opds feed (opds://www.atramenta.net/opds/catalog.atom)

but it doesn't appear inside opds feed

done https://github.com/readium/readium-desktop/pull/639/commits/cac1e6acdd7568b7a506c1aab8ffff29d8301e51